be/domain.c

```diff
@@ -60,11 +60,11 @@ static void be_domain_unlock(struct m0_be_domain *dom)
 	m0_mutex_unlock(&dom->bd_lock);
 }
 
-static int segobj_opt_iterate(struct m0_be_seg         *dict,
-			      const struct m0_be_0type *objtype,
-			      struct m0_buf            *opt,
-			      char                    **suffix,
-			      bool                      begin)
+M0_INTERNAL int m0_be_segobj_opt_iterate(struct m0_be_seg         *dict,
```

Comment:
No need to change this function from "static" to "M0_INTERNAL".
It is used only in this function.

Reply:
it should be static, will update it